backend/src/main/dto/AssignedDTOs.ts

```diff
@@ -0,0 +1,9 @@
+export interface AssignedCreationDTO {
+    taskId?: number,
```

Comment:
You shouldn't have "?" for creation dtos, since these are required properties. This comment applies to all other DTOs file.